SteamKit2/SteamKit2/Steam/Handlers/SteamGameServer/SteamGameServer.cs

```diff
@@ -161,7 +161,7 @@ public void LogOnAnonymous( uint appId = 0 )
 
             logon.Body.client_os_type = ( uint )Utils.GetOSType();
             logon.Body.game_server_app_id = ( int )appId;
-            logon.Body.machine_id = MachineIdUtils.GenerateMachineID();
+            logon.Body.machine_id = HardwareUtils.GenerateMachineID();
```

Comment:
Well I just checked normal logins, and they send an empty MessageObject on windows. Someone in steamworks forum was complaining that steamcmd didn't work last night :laughing: